setup.py

```diff
@@ -67,6 +67,7 @@ def readme():
     entry_points={
         'pytest11': [
             'pysoa_test_plan=pysoa.test.plugins.pytest',
+            'pysoa_test_fixtures=pysoa.test.pytest_fixtures',
```

Comment:
If `pytest_fixtures` is handled like a PyTest plugin, then it should go in `pysoa.test.plugins`. We might need to move `pysoa/test/plugins/pytest.py` to `pysoa/test/plugins/pytest/plans.py` and create `pysoa/test/plugins/pytest/fixtures.py`.

Reply:
cool, will move there